ateam_vision_filter/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ if(BUILD_TESTING)
   target_include_directories(${PROJECT_NAME}_test PRIVATE test src)
   target_link_libraries(${PROJECT_NAME}_test ${PROJECT_NAME})
   set_target_properties(${PROJECT_NAME}_test PROPERTIES CXX_STANDARD 17)
-endif()
+  target_compile_options(${PROJECT_NAME}_test PRIVATE -g)
```

Comment:
Instead of manually adding this kind of flag in CMake, we can just set the build type to `Debug` or `RelWithDebInfo` when we want debug symbols.

```suggestion
```

Reply:
Sorry, I tried to use debug mode in VS Code and must have accidentally added this somehow